src/main/java/inventorysetups/InventorySetupsConfig.java

```diff
@@ -365,11 +365,23 @@ default InventorySetupsShowWornItemsFilterID showWornItemsFilter()
 		return InventorySetupsShowWornItemsFilterID.All;
 	}
 
+	@ConfigItem(
+			keyName = CONFIG_KEY_SECTION_MODE,
+			name = "Submenu in Section mode",
```

Comment:
Name should be like `Show section submenus in worn items`. Is this too long though?

Reply:
It's slightly too long to render without an ellipsis but it seems like the important info still show
![image](https://user-images.githubusercontent.com/93336804/229118094-2c169c62-e69e-4a8c-ac74-be9bbf0f5d66.png)

Reply:
Maybe "Worn Items Section Submenus"?

Reply:
Fits nicely